src/gengowatcher/ui_textual.py

```
@@ -616,13 +616,127 @@ def _render_chart(self) -> Text:
+        "user_session",
+        "user_key",
+        "client_id",
+        "client_secret",
+        "refresh_token",
+        "access_token",
+        "auth_token",
+        "session_cookie",
+        "password",
+        "secret",
+        "token",
+    }
```

Ruff (0.14.14)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)